dockers/docker-fpm-frr/staticd.conf.j2

```diff
@@ -0,0 +1,29 @@
+!
+{% block banner %}
+! =========== Managed by sonic-cfggen DO NOT edit manually! ====================
+! generated by templates/frr/staticd.conf.j2 using config DB data
+! file: staticd.conf
+!
+{% endblock banner %}
+!
+{% block sys_init %}
+hostname {{ DEVICE_METADATA['localhost']['hostname'] }}
+password zebra
+enable password zebra
+{% endblock sys_init %}
+!
```

Comment:
I didn't find anything in staticd which could parse hostname, password and enable password options. Do we need them here?

Reply:
I referred to the conf files generated by "copy running-config startup-config" in vtysh. hostname, password and logging options appear in all of the bgpd.conf, zebra.conf and staticd.conf, but enable password option only appears in zebra.conf and staticd.conf. 
> root@sonic:/# cat /etc/frr/staticd.conf
!
! Zebra configuration saved from vty
!   2019/08/09 09:53:10
!
frr version 7.0.1-sonic
frr defaults traditional
!
hostname sonic
password zebra
enable password zebra
log syslog informational
log facility local4
!
!
!
ip route 0.0.0.0/0 10.11.0.1 200
line vty
!